_widget/src/components/app/component.vue

```diff
@@ -135,7 +136,7 @@ export default {
     },
 
     showRecentlyVisitedArticles () {
-      return this.q.length === 0 && this.recentlyVisitedArticles?.length > 0;
+      return this.q.length === 0 && this.filteredArticles.length === 0 && this.recentlyVisitedArticles?.length > 0;
```

Comment:
This was essentially never showing the URL-matched articles. Now, we show them if they exist. 

You can see your history at any time by clearing the query.